core/string/ustring.cpp

```diff
@@ -2840,6 +2840,13 @@ String String::insert(int p_at_pos, const String &p_string) const {
 	return pre + p_string + post;
 }
 
+String String::erase(int p_pos, int p_chars) const {
+	if (p_chars < 0 || p_pos < 0) {
+		return "";
+	}
```

Comment:
I think an error should be printed before returning `""`, as negative offset or character count clearly sounds like a programming mistake.

Reply:
Shouldn't `p_chars` and `p_pos` be swapped in the conditions?

Reply:
> Shouldn't `p_chars` and `p_pos` be swapped in the macros?

Fixed.